docs/das-cli-users-guide.md

```diff
@@ -0,0 +1,450 @@
+
+- **For the Linux Host Machine**: **Docker** must be installed and running.
+- **For macOS/Windows Client Machines**: **Python 3.10+** is required to run `das-cli`.
+
+---
+
+## Installation
+
+### 1. For Linux Users (Debian-based systems like Ubuntu)
+
+On a Linux machine, you have full flexibility. The recommended method is APT.
+
+#### APT Package (Recommended)
+
+```bash
+# Set up the repository
+sudo bash -c "wget -O - http://45.77.4.33/apt-repo/setup.sh | bash"
+
+# Install the package
+sudo apt install das-toolbox
+
+# Verify the installation
+das-cli --version
+```
+
+**NOTE: Installation an specific version**
+
+To install a lower version of `das-cli` than the current version, you can use the following command:
+
+```bash
+sudo das-cli update-version --version x.y.z
+```
+
+If this command does not work, you can try.
+
+```bash
+sudo apt install --only-upgrade --allow-downgrades das-toolbox=x.y.z
+```
+
+#### From Source
+
+This method is **required for macOS and Windows users** to install the client that sends remote commands. It also works on Linux.
+
+```bash
+# Clone the repository with its submodules
+git clone --recurse-submodules https://github.com/singnet/das-toolbox.git
+cd das-toolbox
+
+# Create and activate a virtual environment
+python3 -m venv env
+source env/bin/activate
+
+# Install dependencies
+cd das-cli/src
+pip3 install -r requirements.txt
+
+# Run the CLI
+python3 das_cli.py --version
+```
+
+#### Binary Release
+
+Download the precompiled binary for your operating system from the [official releases page](https://github.com/singnet/das-toolbox/releases).
+
+### 2. For macOS/Windows Users (Remote Execution)
+
+On these operating systems, you can only use `das-cli` from source to act as a remote client.
+
+```bash
+# Clone the repository with its submodules
+git clone --recurse-submodules https://github.com/singnet/das-toolbox.git
+cd das-toolbox
+
+# Create and activate a virtual environment
+python -m venv env
+env\Scripts\activate.bat
+
+# Install dependencies
+cd das-cli/src
+pip install -r requirements.txt
+pip install windows-curses 
+
+# Run the CLI
+python das_cli.py --version
+```
+
+---
+
+## Quickstart Guides
+
+This section will guide you through the entire process of configuring `das-cli`, launching the necessary services, and loading your first knowledge base.
+
+-   **For Linux users:** Run the commands as shown.
+-   **For macOS/Windows users:** Take each command shown below and append the remote execution flags (`--remote --host <your-host> --user <your-user> --key-file <path-to-your-private-key>`) or (`--remote --host <your-host> --user <your-user> --password <your-password>`).
+
+### Step 1: Configure DAS CLI
+
+First, you must configure the connection details for the services `das-cli` will manage. This is an interactive process.
+
+1. Linux users
+```bash
+das-cli config set
+```
+2. macOS/Windows users. **Choose one of the options below.**
```

Comment:
Linux users may want to setup remote environments as well. You need to re-structure the section names and the way you explain the commands in order to cover this use case too. I'm not asking you to replicate the commands/examples, just changing the way you write the text and section names should be enough.